tests/theia_automation_lsp/robot_suites/__init__.robot

```diff
@@ -0,0 +1,11 @@
+# Copyright (c) 2019 Broadcom.
```

Comment:
delete if no need

Reply:
Files like __init__.robot are needed to be able to run all robot scenarios by one command